src/roll-chromium.ts

```diff
@@ -156,34 +148,69 @@ export async function rollChromium(
 }
 
 export async function rollChromium4(
-  electronBranch: string,
+  electronBranch: {name: string, commit: {sha: string}},
   chromiumVersion: string,
-): Promise<string | null> {
-  d(`triggered for electronBranch=${electronBranch} chromiumVersion=${chromiumVersion}`);
+): Promise<void> {
+  d(`roll triggered triggered for electronBranch=${electronBranch.name} chromiumVersion=${chromiumVersion}`);
   const github = await getOctokit();
-  // Get current SHA of {electronBranch} on electron/electron
-  const electronReference = await github.git.getRef({
+
+  // Look for a pre-existing PR that targets this branch to see if we can update that.
+  const existingPrsForBranch = await github.pulls.list({
+    per_page: 100,
+    base: electronBranch.name,
     owner: 'electron',
     repo: 'electron',
-    ref: `heads/${electronBranch}`,
+    state: 'open',
   });
-  const electronSha = electronReference.data.object.sha;
-  const forkRef = `refs/heads/roller/chromium-${chromiumVersion}-${Date.now()}`;
+  const myPrs = existingPrsForBranch.data.filter((pr) => pr.user.login === PR_USER);
+
+  if (myPrs.length) {
+    // Update the existing PR (s?)
+    for (const pr of myPrs) {
+      d(`found existing PR: #${pr.number}, updating`);
+      await updateDepsFile4(pr.head.ref, chromiumVersion);
+      // TODO: update PR body
+    }
+  } else {
+    d(`no existing PR found, raising a new PR`);
+    // Create a new ref that the PR will point to
+    const electronSha = electronBranch.commit.sha;
+    const branchName = `roller/chromium-${electronBranch.name}-${Date.now()}`;
```

Comment:
Can we get rid of the `Date.now()` hax now?

E.g.
```suggestion
    const branchName = `roller/${electronBranch.name}`;
```

Reply:
Maybe? It wasn't clear to me that there would only ever be one PR per branch but on reflection it seems impossible for there to be more. I'll drop it for now and we can re-add it if it becomes an issue.